### PR TITLE
Updated AWS Lambda Cloudwatch logs collection section.

### PR DESCRIPTION
**Pull request by npande**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/integrations/amazon-aws/lambda.md [hunk 1]
```diff
@@ -143,15 +143,16 @@ This section provides instructions for setting up log and metric collection.
 
 ### Collect Amazon CloudWatch Logs
 
-Sumo supports several methods for collecting Lambda logs from Amazon CloudWatch.
+Sumo supports several methods for collecting Lambda logs from Amazon CloudWatch. You can choose either of them to collect logs.
+- **AWS Kinesis Firehose for Logs**. Configure an [AWS Kinesis Firehose for Logs](https://help.sumologic.com/docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-logs-source/#create-an-aws-kinesis-firehose-for-logssource) (Recommended); or
```
Comment (JainM6) [inline docs/integrations/amazon-aws/lambda.md:147]:
```suggestion
- **AWS Kinesis Firehose for Logs**. Configure an [AWS Kinesis Firehose for Logs](https://help.sumologic.com/docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-logs-source/#create-an-aws-kinesis-firehose-for-logssource) (Recommended).
```

File: docs/integrations/amazon-aws/lambda.md [hunk 2]
```diff
@@ -143,15 +143,16 @@ This section provides instructions for setting up log and metric collection.
...
-* To configure collection without using CloudFormation, see [Collect Amazon CloudWatch Logs using a Lambda Function](/docs/send-data/collect-from-other-data-sources/amazon-cloudwatch-logs/collect-with-lambda-function.md).
-* While configuring the cloud Watch log source, following Field can be added in the source:
-    * Add an **account** field and assign it a value which is a friendly name / alias to your AWS account from which you are collecting logs. This name will appear in the Sumo Logic Explorer View. Logs can be queried via the “account field”.
-    * Add a **region** field and assign it the value of the respective AWS region where the Application Load Balancer exists.
-    * Add an **accountId **field and assign it the value of the respective AWS account id which is being used.
+* While configuring the cloudwatch log source, following Fields can be added in the source:
+    * Add an **account** field and assign it a value which is a friendly name / alias to your AWS account from which you are collecting logs. This name will appear in the Sumo Logic Explorer View. Logs can be queried via the “account" field.
```
Comment (JainM6) [inline docs/integrations/amazon-aws/lambda.md:151]:
```suggestion
    * Add an **account** field and assign it a value that is a friendly name/alias to your AWS account from which you are collecting logs. This name will appear in the Sumo Logic Explorer View. Logs can be queried via the “account" field.
```

File: docs/integrations/amazon-aws/lambda.md [hunk 3]
```diff
@@ -143,15 +143,16 @@ This section provides instructions for setting up log and metric collection.
...
+    * Add a **region** field and assign it the value of the respective AWS region where the Lambda function exists.
+    * Add an **accountId** field and assign it the value of the respective AWS account id which is being used.
```
Comment (JainM6) [inline docs/integrations/amazon-aws/lambda.md:153]:
```suggestion
    * Add an **accountId** field and assign it the value of the respective AWS account ID that is being used.
```

**Review by JainM6 — DISMISSED** [review]
Thanks Nitin for submitting the PR. It looks great. I just found some minor formatting changes.

File: docs/integrations/amazon-aws/lambda.md [hunk 4]
```diff
@@ -143,15 +143,16 @@ This section provides instructions for setting up log and metric collection.
 
 ### Collect Amazon CloudWatch Logs
 
-Sumo supports several methods for collecting Lambda logs from Amazon CloudWatch.
+Sumo supports several methods for collecting Lambda logs from Amazon CloudWatch. You can choose either of them to collect logs.
```
Comment (kimsauce) [inline docs/integrations/amazon-aws/lambda.md:146]:
```suggestion
Sumo Logic supports several methods for collecting Lambda logs from Amazon CloudWatch. You can choose either of them to collect logs.
```

File: docs/integrations/amazon-aws/lambda.md [hunk 5]
```diff
@@ -143,15 +143,16 @@ This section provides instructions for setting up log and metric collection.
...
+- **AWS Kinesis Firehose for Logs**. Configure an [AWS Kinesis Firehose for Logs](https://help.sumologic.com/docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-logs-source/#create-an-aws-kinesis-firehose-for-logssource) (Recommended); or
+- **Lambda Log Forwarder**. Configure a collection of Amazon CloudWatch Logs using our AWS Lambda function using a Sumo Logic provided CloudFormation template, as described in [Amazon CloudWatch Logs](https://help.sumologic.com/docs/send-data/collect-from-other-data-sources/amazon-cloudwatch-logs/) or configure collection without using CloudFormation, see [Collect Amazon CloudWatch Logs using a Lambda Function](https://help.sumologic.com/docs/send-data/collect-from-other-data-sources/amazon-cloudwatch-logs/collect-with-lambda-function/).<br/>
```
Comment (kimsauce) [inline docs/integrations/amazon-aws/lambda.md:148]:
```suggestion
- **Lambda Log Forwarder**. Configure a collection of Amazon CloudWatch Logs using our AWS Lambda function using a Sumo Logic provided CloudFormation template, as described in [Amazon CloudWatch Logs](https://help.sumologic.com/docs/send-data/collect-from-other-data-sources/amazon-cloudwatch-logs/) or configure collection without using CloudFormation, see [Collect Amazon CloudWatch Logs using a Lambda Function](/docs/send-data/collect-from-other-data-sources/amazon-cloudwatch-logs/collect-with-lambda-function/).<br/>
```

**Review by kimsauce — DISMISSED** [review]
minor grammar stuff

File: docs/integrations/amazon-aws/lambda.md [hunk 6]
```diff
@@ -143,15 +143,16 @@ This section provides instructions for setting up log and metric collection.
 
 ### Collect Amazon CloudWatch Logs
 
-Sumo supports several methods for collecting Lambda logs from Amazon CloudWatch.
+Sumo supports several methods for collecting Lambda logs from Amazon CloudWatch. You can choose any of them to collect logs.
+- **AWS Kinesis Firehose for Logs**. Configure an [AWS Kinesis Firehose for Logs](https://help.sumologic.com/docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-logs-source/#create-an-aws-kinesis-firehose-for-logssource) (Recommended).
+- **Lambda Log Forwarder**. Configure a collection of Amazon CloudWatch Logs using our AWS Lambda function using a Sumo Logic provided CloudFormation template, as described in [Amazon CloudWatch Logs](https://help.sumologic.com/docs/send-data/collect-from-other-data-sources/amazon-cloudwatch-logs/) or configure collection without using CloudFormation, see [Collect Amazon CloudWatch Logs using a Lambda Function](/docs/send-data/collect-from-other-data-sources/amazon-cloudwatch-logs/collect-with-lambda-function/).<br/>
```
Comment (kimsauce) [inline docs/integrations/amazon-aws/lambda.md:148]: @npande , we use relative links in Docs

```suggestion
- **AWS Kinesis Firehose for Logs**. Configure an [AWS Kinesis Firehose for Logs](/docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-logs-source/#create-an-aws-kinesis-firehose-for-logssource) (Recommended).
- **Lambda Log Forwarder**. Configure a collection of Amazon CloudWatch Logs using our AWS Lambda function using a Sumo Logic provided CloudFormation template, as described in [Amazon CloudWatch Logs](/docs/send-data/collect-from-other-data-sources/amazon-cloudwatch-logs/) or configure collection without using CloudFormation, see [Collect Amazon CloudWatch Logs using a Lambda Function](/docs/send-data/collect-from-other-data-sources/amazon-cloudwatch-logs/collect-with-lambda-function/).<br/>
```

**Review by kimsauce — APPROVED** [review]

**Pull request merged**